src/material/tooltip/tooltip.ts

```diff
@@ -542,43 +551,78 @@ export class MatTooltip implements OnDestroy, AfterViewInit {
   }
 
   /** Binds the pointer events to the tooltip trigger. */
-  private _setupPointerEvents() {
+  private _setupPointerEnterEventsIfNeeded() {
+    // Optimization: Defer hooking up events if there's no message or the tooltip is disabled.
+    if (this._disabled || !this.message || !this._viewInitialized ||
+        this._passiveListeners.length) {
+      return;
+    }
+
     // The mouse events shouldn't be bound on mobile devices, because they can prevent the
     // first tap from firing its click event or can cause the tooltip to open for clicks.
     if (!this._platform.IOS && !this._platform.ANDROID) {
       this._passiveListeners
-        .set('mouseenter', () => this.show())
-        .set('mouseleave', () => this.hide());
+          .push(['mouseenter', () => {
+            this._setupPointerExitEventsIfNeeded();
+            this.show();
+          }]);
+    } else if (this.touchGestures !== 'off') {
+      this._disableNativeGesturesIfNecessary();
+
+      this._passiveListeners
+          .push(['touchstart', () => {
+            // Note that it's important that we don't `preventDefault` here,
+            // because it can prevent click events from firing on the element.
+            this._setupPointerExitEventsIfNeeded();
+            clearTimeout(this._touchstartTimeout);
+            this._touchstartTimeout = setTimeout(() => this.show(), LONGPRESS_DELAY);
+          }]);
+    }
+
+    this._hookupListeners(this._passiveListeners);
+  }
+
+  private _setupPointerExitEventsIfNeeded() {
+    if (this._pointerExitEventsInitialized) {
+      return;
+    }
+    this._pointerExitEventsInitialized = true;
+
+    const exitListeners: (readonly [string, EventListenerOrEventListenerObject])[] = [];
+    if (!this._platform.IOS && !this._platform.ANDROID) {
```

Comment:
I'd extract the platform check to a private method to avoid duplicating it, something like `_platformSupportsMouseEvents`